codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/middleware/factories/HostPrefixMiddlewareFactory.java

```diff
@@ -37,6 +37,7 @@ public static Middleware build(GenerationContext context) {
         ClientConfig disableHostPrefix = ClientConfig.builder()
                 .name("disable_host_prefix")
                 .type("Boolean")
+                .rbsType("bool")
```

Comment:
We have some special logic around the "Boolean" "type" already - I wonder if its worth adding a special case in the builder for this to just automatically set the rbsType?

Reply:
Maybe? Can think about other improvements too.. we definitely need an override for this anyway. Arguably we should have .type("TrueClass, FalseClass") (it could even be documented this way in Yard) and also "Boolean" as docType.